components/ui/InfiniteScroll.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <section>
```

Comment:
I'd argue that we should get rid of the `section` wrapper here since the `InfiniteScroll` component looks more like a reusable UI element / factory, than a [**standalone section**](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section).

As per [spec](https://html.spec.whatwg.org/multipage/sections.html#the-section-element):

> The section element represents a generic section of a document or application. A section, in this context, is a thematic grouping of content, typically with a heading.

and

> The section element is not a generic container element. When an element is needed only for styling purposes or as a convenience for scripting, authors are encouraged to use the div element instead. A general rule is that the section element is appropriate only if the element's contents would be listed explicitly in the document's outline.

In this context, I don't believe that this block should be semantically considered a "section". The section would rather be the parent element that includes the cards with infinite scrolling, but also the filter and the "Meet the Advocates" headline.